tutorials/cloudsdk-js-vdm-delete/cloudsdk-js-vdm-delete.md

```diff
@@ -141,10 +141,40 @@ function deleteBusinessPartnerAddress(businessPartnerId: string, addressId: stri
     });
 }
 ```
+
+As of version `1.7.0` of the SAP Cloud SDK, you can alternatively pass an entity to the `delete` method instead of the entity's key fields, like this:
+
+```JavaScript / TypeScript
+function deleteBusinessPartnerAddress(address: BusinessPartnerAddress): Promise<void> {
+  return BusinessPartnerAddress.requestBuilder()
+    .delete(address)
+    .execute({
+      url: 'https://my.s4hana.ondemand.com/'
+    });
+}
+```
+
+This has the advantage that the version identifier of the entity is set automatically. When going with the first approach, you would have to manually supply the version identifier like this (notice the changed signature of `deleteBusinessPartnerAddress`):
```

Comment:
I think this does not warrant a code snippet in the tutorial, which we should keep short and to-the-point. Keep the explanation (perhaps as one of the decision hints above) and maybe refer to the JSDoc for setVersionIdentifier.